Makefile

```diff
@@ -117,6 +117,7 @@ clean:
 
 prepare:
 	@mkdir -p bin obj debug dep
+	pip install -r requirements.pip
```

Comment:
This is the only actual change in this file ;)

Reply:
then why don't you remove the other changes in the commit? they are just noise

Reply:
@knocte The other changes are removing the "noise" left here through the time. Unfortunately atm I don't have any editor set up to leave that "noise" in files I edit. If You can't use this PR because of that feel free to reuse the change We discuss here in Your own, new PR.

Reply:
I'm not the maintainer of this project, I just commented to help you. Maybe your PR didn't get merged because of this? You don't make style changes in the same commit as where you're fixing a problem, if you want your PR merged. Read this http://tirania.org/blog/archive/2010/Dec-31.html

Reply:
Anyway there are now conflicts, so you would need to rebase.

Reply:
RE @knocte -- https://github.com/LedgerHQ/blue-app-ssh-agent/pull/4#discussion_r145144799 : 
Ty, for advise and link. I'll definitely read it, it looks interesting.

Checking the conflict probably tonight

Reply:
Actually the reason for the need of a rebase is because of including style changes ;) if they hadn't been there, then this PR wouldn't have had conflicts ;)